spec/mailers/previews/problem_order_mailer_preview.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ProblemOrderMailerPreview < ActionMailer::Preview
+
+  def notify_user
+    order_detail = NUCore::Database.random(Reservation.user.joins(:order_detail).merge(OrderDetail.complete)).order_detail
```

Comment:
Instead of `random`, why don’t we move these previews to use `first` and get rid of the method entirely? That way they’d show a predictable email for previewing, too.

Reply:
I personally like the non-predictability here. It's saved me from missing some null pointers and whatnot. Not a big deal if you feel strongly about it.

Reply:
I don’t care very much about it, so fine by me! (I would tend towards getting rid of the method on `NUCore::Database` myself, but this is okay, too)